SECURITY.md

```diff
@@ -1,20 +1,102 @@
 # Security Policy
 
-## Latest Versions
-
-We advise users to run the most recent mainline or stable release of nginx.
+This document provides an overview of security concerns related to nginx
+deployments, focusing on confidentiality, integrity, availability, and the
+implications of configurations and misconfigurations.
 
 ## Reporting a Vulnerability
 
-Please report any vulnerabilities via one of the following methods
-(in order of preference):
+Please report any vulnerabilities via one of the following methods (in order of
+preference):
 
-1. [Report a vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability)
-within this repository. We are using the Github workflow that allows us to
-manage vulnerabilities in a private manner and to interact with reporters
-securely.
+1. [Report a vulnerability](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing-information-about-vulnerabilities/privately-reporting-a-security-vulnerability) within this
+repository. We are using the GitHub workflow that allows us to manage
+vulnerabilities in a private manner and interact with reporters securely.
 
 2. [Report directly to F5](https://www.f5.com/services/support/report-a-vulnerability).
 
-3. Report via email to security-alert@nginx.org.
-This method will be deprecated in the future.
+3. Report via email to security-alert@nginx.org. This method will be deprecated
```

Comment:
While the scope of this PR is to update the currently security policy, does it make sense to remove this method to report stuff and just keep reporters focussed on the first two.

Reply:
For now we'd like to keep it intact but will remove it in the near future.

Reply:
👍